README.md

```diff
@@ -1 +1,124 @@
-# capston-project-js
\ No newline at end of file
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse JavaScript Capstone Project</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+# 📖 [JavaScript-Group-Capstone] <a name="about-project"></a>
+
+
+**[JavaScript-Group-Capstone]** is a basic website that displays a number of meals. A user can like, or comment on a meal they choose. We will achieve that by using JavaScript to dynamically modify the DOM.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- Html
+- css
+- Javascript
+- Visual Studio Code
+
+
+### Key Features <a name="key-features"></a>
+
+- **[Home_Page]**
+- **[Comments_Popup]**
+- **[Reservation_Popup]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+1. clone the project from link: https://github.com/girma336/capston-project-js.git by using git clone (link).
+2. open it on ur vs code and start using it. 
+
+
+### Usage
+
+To run the project, execute the following command:
+
+npm start
+
+### Run tests
+
+To run tests, run the following command:
+
+npm test
+
+### Deployment
+
+You can deploy this project using:
+
+npm i -D gh-pages
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@girma336](https://github.com/girma336)
```

Comment:
[OPTIONAL] This is suppose to be a group work but I see only one author. Kindly add the name of the other author. :pray:

Reply:
hi @jerryowusu 
- this is solo project their is not other coding partner

Reply:
I reviewed your project as a group collaborative project, so if you are claiming a solo project then you have to bring evidence of student success permitting you to do it alone to the next reviewer else your project will be invalidated because you need to do it with your current partner for this week